qt/MainWindow.cc

```diff
@@ -285,7 +285,7 @@ MainWindow::MainWindow(Session& session, Prefs& prefs, TorrentModel& model, bool
     menu->addSeparator();
     menu->addAction(ui_.action_Quit);
     tray_icon_.setContextMenu(menu);
-    tray_icon_.setIcon(QIcon::fromTheme(QStringLiteral("transmission-tray-icon"), QApplication::windowIcon()));
+    tray_icon_.setIcon(QIcon::fromTheme(QStringLiteral(":icons/transmission-tray.svg"), QApplication::windowIcon()));
```

Comment:
Ditto re. being unable to override via a theme due to icon name not being a real name but a resource file path.

Reply:
Yes, and it's a problem i struggle with, between wanting transmission-qt to look and feel coherent in-house before it goes out the door, and leaving room for whatever users want once it's out, but without the awful combinations i'm seeing used by some linux themes (GTK icon and macOS tray icon for example). But the changes, i think, definitely make things better for macOS and Windows users.

Reply:
Altruistically, i'm in favour of theming. But i'm not in favour of the choices i'm seeing theme-makers make to Transmission's menu bar where everything appears to be cobbled from multiple icon sources.